src/azure-openai/src/chatCompletion/AzureOpenAIChatCompletionService.ts

```diff
@@ -26,17 +26,19 @@ export class AzureOpenAIChatCompletionService implements ChatCompletionService {
    * @param params.provider Azure OpenAI provider (optional).
    */
   public constructor({
-    model,
+    deploymentName,
```

Comment:
We should update the comments for the constructor to match the parameters with these changes.